setup/www/resources/config/nodejs.org

```diff
@@ -40,7 +40,11 @@ server {
     default_type text/plain;
     index index.html;
 
+    error_page 404 @localized_404;
+
     location / {
+        try_files $uri $uri/ @english_fallback;
```

Comment:
fwiw this directive is only processing `^/dist/?.*$` and `^/.*\.json$` so I don't think it needs to be localized because even the .json are only in the docs which are not translated (afaik). Also, the whole `http` section is going away _soon_.